### DISCUSSION TITLE
Wrong output while connecting

### DISCUSSION THREAD

**Discussion by garak** [deployphp/deployer#2273] [Help needed] [answered]
I'm reluctant to directly open an issue on this, since it's something that I observe randomly.
Let's try here.

Sometimes, instead of seeing the "dice" rotate, I see a bunch of lines, one under the another. It's like something on the terminal gone wrong.
Like (output redacted, it's much more):
```
⠹ connect production                                                                                                                                        
⠹ connect production                                                                                                                                        
⠹ connect production                                                                                                                                        
⠹ connect production                                                                                                                                        
⠹ connect production                                                                                                                                        
⠹ connect production                                                                                                                                        
⠹ connect production                                                                                                                                        
⠹ connect production                                                                                                                                       
⠹ connect production                                                                                                                                        
⠸ connect production                                                                                                                                        
⠸ connect production                                                                                                                                      
```

Using deployer 7-beta4

**Comment by antonmedv** [comment]
What’s your terminal? Os? Not every time?

**Reply by garak** [reply]
> What’s your terminal? Os? Not every time?

Ubuntu 20.10, default terminal app.
Sadly, this is not happening every time. Maybe it can depend on remote machine, too?

**Reply by antonmedv** [reply]
Can’t tell why. I think something in printing \n

And connect task using this \r connect

**Reply by garak** [reply]
Is there a way to debug it? Maybe printing newline characters somewhere?

**Reply by antonmedv** [reply]
Print it to some editor with forced --ansi and see what chars a printer exactly.

**Reply by antonmedv** [reply]
```
dep deploy --ansi | some unicode viewer
```

**Comment by antonmedv** [comment]
I think i got idea. Do you change term size?. Or try other

**Comment by martijngastkemper** [comment]
I'm seeing this effect when running Deployer on Github actions:

<img width="526" alt="Screenshot 2020-11-07 at 21 13 54" src="https://user-images.githubusercontent.com/250662/98450595-2b1de300-213e-11eb-90b8-d93d9ad91e2c.png">

This goes on for several thousand lines.

**Reply by garak** [reply]
Same here. Maybe we could add an option to avoid such "moving" feedback?

**Comment by antonmedv** [comment]
I think i get it. Will try to release new version with a fix today.

**Comment by garak** [comment, marked as answer]
It looks like the last beta (beta5) solved this (at least, I'm not experienced it anymore)

**Reply by antonmedv** [reply]
There was an optimisation what caused this. Deleted it.
